MBIPS/MBIP-4.md

```diff
@@ -38,17 +38,29 @@ A transaction with not enough `gasPrice` to compensate for the gas and storage w
 
 ### `storageBaseFee`
 
-The storage base fee is the cost of storing 1 byte of data in the chain state.
+The storage base fee is the cost of storing 1 byte of data in the chain state. One straightforward
+approach to dynamically adjusting this would be to use the same formula used in
```

Comment:
Typo in adjusting no ?

Reply:
What do you mean? Gramatically?

Reply:
As a non-english speaker I would have written:
`One straightforward approach to dynamically adjust this`

Reply:
https://quillbot.com/grammar-check thinks that "adjusting" is correct :) I probably could have explained why when I was younger :shrug: